BtConnectorLib/btconnectorlib2/src/main/java/org/thaliproject/p2p/btconnectorlib/WifiServiceSearcher.java

```diff
@@ -274,8 +275,13 @@ public void onFailure(int reason) {
 
     private void stopDiscovery() {
         p2p.clearServiceRequests(channel, new WifiP2pManager.ActionListener() {
-            public void onSuccess() {debug_print("Cleared service requests");}
-            public void onFailure(int reason) {debug_print("Clearing service requests failed, error code " + reason);}
+            public void onSuccess() {
+                debug_print("Cleared service requests");
+            }
+
+            public void onFailure(int reason) {
+                debug_print("Clearing service requests failed, error code " + reason);
+            }
         });
     }
     private void debug_print(String buffer) {
```

Comment:
I suppose the way I'm using it is recommended: http://stackoverflow.com/questions/20660628/android-how-to-use-switch-case-with-intents-received-from-broadcasts

Also I'm not interested for any other action, thus having handling for it would not make sense to me.